node-operators/networks/collator.md

```diff
@@ -24,13 +24,18 @@ This guide will take you through the following steps:
  - **[Accounts and staking requirements](#accounts-and-staking-requirements)** — goes through the process of getting your account set up and bond tokens to become a candidate
  - **[Generate session keys](#generate-session-keys)** — explains how to generate session keys, used to map your author ID with your H160 account
  - **[Map author ID to your account](#map-author-id-to-your-account)** — outlines the steps to map your public session key to your H160 account, where block rewards will be paid to
+ - **[Join the Discord](#join-discord)** — introduces the best way to keep up with important collator updates, as well as where to go to if you have any questions
 
 ## Technical Requirements {: #technical-requirements } 
 
 From a technical perspective, collators must meet the following requirements:
 
  - Have a full node running with the collation options. To do so, follow the [Run a Node](/node-operators/networks/run-a-node/overview/) tutorial, considering the specific code snippets for collators
 
+!!! attention
+    Running a **collator** node has higher CPU requirements than the ones provided in the above tutorial. In order for your collator node to be able to keep up with a high transaction throughput, a CPU with high clock speed and single thread performance is important. We have tested and seen good results with the following CPUs: Intel Xeon E-2386/2388 and Ryzen 9 5950x/5900x. As most cloud providers focus on multithreading rather than single-thread performance, using a bare-metal provider is recommended.
+    Running your collator node in Docker is also not recommended, as it will have a significant impact in performance.
```

Comment:
just want to double check is this intended to be part of the note? because with the 4 spaces in front it will be

Reply:
Yes, both paragraphs are supposed to be part of the node

Reply:
This is just a temporary note, as we plan on rewriting the collators page a bit to include more detailed information, but I'll see if I can emphasize that a bit more